docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -417,6 +422,9 @@ Return low-level information on the container `id`
 				"Type": "json-file"
 			},
 			"SecurityOpt": null,
+			"Sysctls": {
+			        "net.ipv4.ip_forward": "1"
+                        },
```

Comment:
just a nit; does this curly bracket align properly? Could be GitHub isn't rendering it properly